components/HomeSection/HomeSection.module.scss

```diff
@@ -4,38 +4,52 @@
   position: relative;
 
   h2 {
-    font-size: 2.5rem;
-    font-weight: 600;
-    letter-spacing: -2px;
-    margin: 0 auto;
-    max-width: 1080px;
-    padding: 0 1rem 1rem;
     position: relative;
     z-index: 2;
+    max-width: 67.5rem;
+    margin: 0 auto 1rem;
+    padding: 0 2rem;
+    letter-spacing: -0.125rem;
+
+    font: {
+      size: 2.5rem;
+      weight: 600;
+    }
```

Comment:
While I appreciate the ability to nest rules in this way, it makes using automatic alphabetizing more difficult and I would prefer we do not do this.